main.c

```diff
@@ -65,9 +68,32 @@ static void handle_sequence(int fd, uint64_t seq, uint64_t ns, uint64_t data) {
 	monitor_crtc(fd, crtc);
 }
 
+static const char usage[] =
+	"Usage: drm_monitor [options...]\n"
+	"\n"
+	"  -d              Specify DRM device number (default 0).\n"
```

Comment:
Nice! Can we make this take an absolute path instead, so that it works with `/dev/dri/by-path` paths and render nodes too?

Reply:
DoneDone